### PR TITLE
28 faq spork page

### PR DESCRIPTION
**Pull request by neilZon**
**Description**
Adds the Spork FAQ Page

closes: #28 

Note: Footer component needs small fix to restyling, will open and tag pr here

### CODE REVIEW

**Review by 10thfloor — CHANGES_REQUESTED** [review]

File: libs/design-system/src/lib/Pages/SporkFAQPage/index.tsx [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+      'Lorem ipsum dolor sit amet. Et officia Quis sed vero corrupti hic fuga asperiores? Qui provident dolor hic pariatur deserunt cum mollitia rerum et tempore sint non fugiat dolor a molestiae deserunt.\n\nAut cumque internos et accusantium Quis aut obcaecati minus vel esse dolores? Sed molestiae unde a nulla amet et debitis laborum aut quidem tempora. Id fugit quia id quia distinctio in minima internos quo laboriosam possimus.\n\nCum optio autem ut velit rerum sed culpa omnis ut deleniti deleniti. Et amet quos labore quia ut ullam dolor et consequatur sint! Ut minus nostrum ea eius voluptates quo minima itaque At repellendus saepe et quasi vitae sit quia illo eos cumque omnis.',
+  },
+];
+
+export function SporkFAQPage() {
+  return (
+    <main className="bg-zinc-50 px-4 pb-48 dark:bg-black md:bg-white md:px-0 md:pt-16">
+      <div className="grid grid-cols-1 items-start pb-6 md:flex md:pb-12">
+        <div className="order-2 grow">
+          <h1 className="pb-14 font-display text-[44px] font-bold leading-snug md:pb-6 md:font-sans md:text-4xl">
```
[the remark this answers is not in the document]
Comment (neilZon) [reply libs/design-system/src/lib/Pages/SporkFAQPage/index.tsx:37]: On the Figma doc the font size for this header is 2.75 rem or 44px but the type scale we have in the tailwind config jumps from  `'4xl': ['2rem', { lineHeight: '2.25rem' }]` to `'5xl': ['3.5rem', { lineHeight: '1' }]` which either look too big or too small for the mobile screens compared to the styling

[the remark this answers is not in the document]
Comment (alse) [reply libs/design-system/src/lib/Pages/SporkFAQPage/index.tsx:37]: @neilZon I'm seeing 32px on https://www.figma.com/file/704SXr4HrUbIIkXYena62Q/Flow-Dev-Portal-(External)?node-id=1495%3A60231 , which would match the text-4xl size

File: libs/design-system/src/lib/Pages/SporkFAQPage/index.tsx [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+            updatedDate="23/3/2022"
+            authorIcon="https://avatars.githubusercontent.com/u/62387156?s=64&v=4"
+            authorName="@maxxP"
+            otherAuthorsCount={12}
+            readMinutes={4}
+            difficulty="Beginners"
+          />
+        </div>
+        <div className="order-1 md:order-2">
+          <button className="mt-2 flex items-center rounded-lg border-primary-blue py-4 text-primary-blue dark:border-primary-purple dark:text-primary-purple md:m-0 md:border md:px-5">
```
Comment (alse) [inline libs/design-system/src/lib/Pages/SporkFAQPage/index.tsx:51]: @neilZon I updated the Button component to use left/right icons, you can use it like this after rebasing from main:

```
      <Button variant="secondary" prev={true}>
          Back to Network
      </Button>
      ```

File: libs/design-system/src/lib/Pages/SporkFAQPage/index.tsx (hunk 1 above)
[the remark this answers is not in the document]
Comment (neilZon) [reply libs/design-system/src/lib/Pages/SporkFAQPage/index.tsx:37]: @alse I have that for the medium screen styling but on the mobile screen the font size changes https://www.figma.com/file/704SXr4HrUbIIkXYena62Q/Flow-Dev-Portal-(External)?node-id=934%3A30763

[the remark this answers is not in the document]
Comment (alse) [reply libs/design-system/src/lib/Pages/SporkFAQPage/index.tsx:37]: Ah cool, I think we can use 4xl on mobile for now to match the other page headers

**Review by alse — APPROVED** [review]

**Pull request merged**